apps/site/next.dynamic.constants.mjs

```diff
@@ -29,6 +32,14 @@ export const IGNORED_ROUTES = [
  * @type {Map<string, import('./types').Layouts>} A Map of pathname and Layout Name
  */
 export const DYNAMIC_ROUTES = new Map([
+  // Creates dynamic routes for downloads archive pages for each version
+  // (e.g., /download/v18.20.8, /download/v20.19.2)
+  ...provideReleaseData()
```

Comment:
My fear is that this will blow static builds in a monstruous way.

Reply:
Do you have a number of how many more routes this generates?

Reply:
Currently, there are 606 pages for the default locale alone. Should all of them be included in the static build, or should we selectively build only the ones that are most frequently accessed (lts, current, or last 6 version)?

Reply:
You can do the following on the ignore list:

- locale isn't defaultLocale and paths starts with /download/archive

Reply:
I wonder how long the static build takes in this branch?

Reply:
> Speed seems actually quite ok. Btw seeing this error on the build "Error: Cannot apply utility class `after:md:-left-1/2` because the `md` variant does not exist."

Rebase, it's been fixed